src/ResourceManagement/ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry/generate.cmd

```diff
@@ -4,9 +4,9 @@
 ::
 
 @echo off
-set autoRestVersion=0.17.0-Nightly20161012
+set autoRestVersion=1.0.0-Nightly20170212
 if  "%1" == "" (
-    set specFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json"
+    set specFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-containerregistry/2017-03-01/swagger/containerregistry.json"
```

Comment:
@djyou can you change this URL to point at the commit (which I believe would be [this URL](https://raw.githubusercontent.com/Azure/azure-rest-api-specs/3b0b26b4b6e3bc5e7cf3610b0866d310abb5b814/arm-containerregistry/2017-03-01/swagger/containerregistry.json)) instead of the path in the rest-api-specs repo?